controllers/databaseclusterbackup_controller.go

```diff
@@ -115,6 +115,12 @@ func (r *DatabaseClusterBackupReconciler) Reconcile(ctx context.Context, req ctr
 		}
 		return reconcile.Result{}, err
 	}
+
+	if (backup.HasSucceeded() || backup.HasFailed()) &&
```

Comment:
I just remembered something...
If I'm not mistaken I think I've seen in the past the status move temporarily to an error state and then getting back to in progress and eventually succeed. I just not sure in which resource this was, DBC, DBB or DBR... :thinking:

Reply:
I'm now slightly worried about having added the `backup.HasFailed()`. I need to test this thoroughly...

Reply:
I can't be 100% sure, but I think it was the DBRestore object..

Reply:
Yeah I think you're right. Let's not block the PR for this, I'll do some more tests to confirm...